tests/integration/test_settings_constraints_distributed/test.py

```diff
@@ -100,44 +100,44 @@ def test_select_clamps_settings():
     # Check that shards clamp passed settings.
     query = "SELECT hostName() as host, name, value FROM shard_settings WHERE name = 'max_memory_usage' OR name = 'readonly' ORDER BY host, name, value"
     assert (
-        distributed.query(query) == "node1\tmax_memory_usage\t99999999\n"
+        distributed.query(query) == "node1\tmax_memory_usage\t50000000\n"
```

Comment:
That's ok. The shards in the test used to clamp the value [10000000000](https://github.com/ClickHouse/ClickHouse/pull/35921/files#diff-9bba202fed6897e19d2f2cce6237e0a8e2c79ac70b8d8b8537b330370f214924L10) got from the initiator of the distribited query to the shard's allowed range [11111111..99999999](https://github.com/ClickHouse/ClickHouse/blob/46fa87b14e5e8f3b7f4107157f5a33ef351ab0d2/tests/integration/test_settings_constraints_distributed/test.py#L73), but now with your changes the default profile doesn't change `max_memory_usage` anymore so the shard uses its own value for `memory usage` which is [50000000](https://github.com/ClickHouse/ClickHouse/blob/46fa87b14e5e8f3b7f4107157f5a33ef351ab0d2/tests/integration/test_settings_constraints_distributed/test.py#L73) instead of clamping.